Documentation/project-docs/developer-guide.md

```diff
@@ -3,6 +3,12 @@ Developer Guide
 
 This guide provides instructions (mostly as links) on how to build the repo and implement improvements. It will expand over time.
 
+Dev Workflow
+===============
+
+The [dev workflow] (../building/dev-workflow.md) describes the development process to follow when working in our repo. 
```

Comment:
Consider writing "this repo" instead of "our repo", as I think it sounds inclusive. This applies to the introduction to dev-workflow.md also.